rpc/src/rpc.rs

```diff
@@ -8082,8 +8100,30 @@ pub mod tests {
                 .to_string(),
         );
         assert_eq!(
-            sanitize_transaction(unsanitary_versioned_tx).unwrap_err(),
+            sanitize_transaction(unsanitary_versioned_tx, &DisabledAddressLoader).unwrap_err(),
             expect58
         );
     }
+
+    #[test]
```

Comment:
I know it's a bit of a pita to construct, but would be nice to have a test case demonstrating unsupported tx error for `getBlock` to make the effect of one v0 tx clear.